scripts/ripe-atlas

```diff
@@ -59,11 +65,55 @@ class RipeAtlas(object):
             return "dns"
         return shortcut
 
-    def main(self):
-
-        self._setup_command()
-
-        cmd_cls = Command.load_command_class(self.command)
+    def autocomplete(self):
+        """
+        This function and it's is highly inspired from Django's own autocomplete
+        manage.py. For more documentation check
+        https://github.com/django/django/blob/1.9.4/django/core/management/__init__.py#L198-L270
+        """
+
+        def print_options(options, curr):
+            """
+            Prints matching with current word available autocomplete options
+            in a formatted way to look good on bash.
+            """
+            sys.stdout.write(' '.join(sorted(filter(lambda x: x.startswith(curr), options))))
+
+        # If we are not autocompleting continue as normal
+        if 'RIPE_ATLAS_AUTO_COMPLETE' not in os.environ:
+            return
+
+        cwords = os.environ['COMP_WORDS'].split()[1:]
```

Comment:
It will be unless complete function is broken. 
The two lines above make sure that we are not going to fetch _COMP_WORDS_ unless is called within bash complete function.

I am going to close this PR and create a new one as soon as I fix problem with tests and travis.